src/Makefile

```diff
@@ -12,7 +12,9 @@ LINKFLAGS = -I $(CAMLP5D)
 include $(ROOT)/tools/Makefile.common
 
 compilation.ml: always
-	echo "Util.compilation_time_hook.val := \" ($$(date "+%Y-%m-%d %T %Z"))\";" > $@
+	echo "Util.compilation_time_hook.val := \" $$(date "+%Y-%m-%d %T %Z")\";" > $@
+	echo "Util.commit_nbr_hook.val := \"$$(git log -1 | grep -x 'commit .*' | cut -c 8-14)\";" >> $@
```

Comment:
ha ha ! best answer ever ! :)
it gives you the commit number and the commit date in a nice and better way